game/data/doctrine.py

```diff
@@ -32,72 +32,146 @@ def from_dict(data: dict[str, float]) -> GroundUnitProcurementRatios:
         return GroundUnitProcurementRatios(r)
 
 
-@dataclass(frozen=True)
-class Doctrine:
-    name: str
-
-    cas: bool
-    cap: bool
-    sead: bool
-    strike: bool
-    antiship: bool
+@dataclass
```

Comment:
(non-blocking, can deal with it later)

Might as well make this `frozen=True`. Aside from being less bug prone, it allows dataclass to autogenerate the `__eq__` and `__hash__` methods.

Reply:
Thanks for reviewing this PR, I'll add this the next time I touch the Doctrine class.